arangod/Replication/ReplicationApplier.cpp

```diff
@@ -26,6 +26,7 @@
 #include "Basics/Mutex.h"
 #include "Basics/MutexLocker.h"
 #include "Basics/ReadLocker.h"
+#include "Basics/StringUtils.h"
```

Comment:
whats the include for

Reply:
I had to add this so the branch actually compiled.
I removed an include of the StringUtils and a few other files from a central header file, which to led lots of compile errors. So I had to include the StringUtils in all the places where they are actually needed.
But it seems afterwards I removed the code from ReplicationApplier.cpp that was using the StringUtils functionality. So I will remove the include here now.

Reply:
Correcting myself: the StringUtils functionality is still used in the ReplicationApplier, so the include must stay there in order to compile things.